admin_ui_support/composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "require": {
         "php": "^5.5.9 || ^7.0",
-        "drupal/jsonapi": "^1.22",
+        "drupal/jsonapi": "2.0.0-beta2",
```

Comment:
+1 for pointing to a specific release. this helps to not just break stuff

Reply:
yay!